coss/home/models.py

```diff
@@ -22,10 +22,18 @@ class AboutSnippet(models.Model):
     header_text = fields.RichTextField(blank=True)
     body_title = models.CharField(blank=True, default='', max_length=40)
     body_text = fields.RichTextField(blank=True)
+    body_image = models.ForeignKey(
+        'wagtailimages.Image',
+        null=True,
+        blank=True,
+        on_delete=models.SET_NULL,
+        related_name='+'
+    )
     info_title = models.CharField(blank=True, default='', max_length=40)
     info_text = fields.RichTextField(blank=True)
+    details_title = models.CharField(blank=True, default='', max_length=40)
```

Comment:
There is another `details` section in the streamfield? Are the two related? If yes, this shouldn't part of the snippet page. If not, it's a bit confusing to have details in two fields.

Reply:
Yes, good point. Since the rest of the details block is also generic and not specific to OS Clubs, I moved it inside that Snippet. Now it's less confusing for a Content creator.

![screenshot-2017-9-6 wagtail - editing about snippet - test](https://user-images.githubusercontent.com/939357/30113226-a6c97342-931c-11e7-8c73-1390de81f94c.png)